overview_tab/map_graph.py

```diff
@@ -4,25 +4,26 @@
 
 FAMILY = "PT Sans"
 
-def data_processing_for_graph(df, postal_code_df):
+def data_processing_for_graph(df, postal_code_df, threshold):
```

Comment:
maybe we can call this variable map_threshold throughout the entire project? in case we have other thresholds later

Reply:
Ok done